temfpy/integration.py

```diff
@@ -317,7 +317,7 @@ def oscillatory(x, a, b):
     a : array_like
         Weight vector with dimension :math:`p`.
     b : int
-        Scale value that increases influences the location of the oscillatory.

```

Comment:
I wasn't sure how you would like to describe the scale value b. I removed 'increases' and just left 'influences'. However, this is fairly non-specific. If you would prefer more precise language, I can make further changes as well such as specifying _how_ increasing/decreasing b affects the location. 

Also, the word 'oscillatory' is an adjective and therefore is not suitable here. Was your intention to write 'oscillations' or to describe a noun?